pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -502,3 +505,69 @@ func checkForPodSecondaryNetworkAttachement(pod *corev1.Pod) (string, bool) {
 		return netObj, false
 	}
 }
+
+// RestoreSecondaryNetwork restore cniCache and pod interface when agent restart.
+func (pc *PodController) RestoreSecondaryNetwork(ovsBridgeClient ovsconfig.OVSBridgeClient, ifaceStore interfacestore.InterfaceStore) {
+	err := InitializeInterfaceStore(ovsBridgeClient, ifaceStore)
+	if err != nil {
+		klog.ErrorS(err, "Failed to initialize the secondary bridge interface store")
+	}
+
+	knownInterfaces := ifaceStore.GetInterfacesByType(interfacestore.ContainerInterface)
+	for _, containerConfig := range knownInterfaces {
+		Config, exists := ifaceStore.GetContainerInterface(containerConfig.ContainerID)
```

Comment:
I am a bit confused here: `knownInterfaces` is read from interface store, and you are loop with `knownInterfaces`, then in what case `exists` would be false since you still check it in the interfacestore?

I guess here you should compare the `knownInterfaces` with the Pods existing in the kube-apiserver, and remove the stale ports from both OVSDB and interfacestore?